src/fs/migration.ts

```diff
@@ -36,7 +34,10 @@ export const parseMigrationFromUrl = (
   try {
     if (urlPart) {
       const bytes = toByteArray(urlPart);
-      const json = JSON.parse(LZMA.decompress(bytes));
+      const json = JSON.parse(
+        // FIXME, reinstate decompression with a working module.
```

Comment:
Needs an alternative lib or some way to import this old-style module.

Reply:
micro:bit classroom manages this but uses a different entry point (the worker one) and copes with different behaviour in dev vs build. Can we find an alternative?